app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ReservationsController < ApplicationController
+  def index
+    reservations = Reservation.all
+    render json: reservations
+  end
```

Comment:
Hi @Dachrono Great job implementing the index and delete actions for reservations. Please consider, that for this part of the app, we should present the user reservations list, so you will need the user as part of the index